### PR TITLE
Integrate grabbing (re-built)

### PR DESCRIPTION
**Pull request by Casper-Oakley**
### Description
(Re-make for https://github.com/FRUK-Simulator/Simulator/pull/126)

Add a block for sending grab opens/closes to the robot. Based off of the WIP https://github.com/FRUK-Simulator/SimulatorCore/pull/127 - this PR is subject to change from that. Have only put in one for inputs, and have left out checking for now. Will put that in with this PR shortly

### Dependencies

https://github.com/FRUK-Simulator/SimulatorCore/pull/127 - Grabber functionality in simulator

### CODE REVIEW

File: src/RobotSimulator/StdWorldBuilder.ts [hunk 1]
```diff
@@ -74,6 +74,30 @@ export class StdWorldBuilder {
...
 
     const robotSpec = robotBuilder.generateSpec();
+
+    robotSpec.mechanisms = [
+      {
+        type: "gripper-mechanism",
+        ioMap: [
+          {
+            id: "grab",
+            channel: 0,
```
Comment (JamesWP) [inline src/RobotSimulator/StdWorldBuilder.ts:84]: I think some other sensors might conflict with these values. but i might be wrong there.

Comment (Casper-Oakley) [reply src/RobotSimulator/StdWorldBuilder.ts:84]: Checking now. I had assumed that they were different channels types, but will step through on the simulator core side. Worst comes to worst ill just bump it up to unique numbers

Comment (Casper-Oakley) [reply src/RobotSimulator/StdWorldBuilder.ts:84]: Looks like they are considered separate systems, and don't conflict. Want me to increment them anyway so they're unique for the end user or just leave as is?

**Comment by JamesWP** [comment]
Either or, (ill approve now) but if its not changed then add a comment about that

**Review by JamesWP — APPROVED** [review]
LGTM

**Comment by Casper-Oakley** [comment]
Will leave as is and come back to it if there's an issue

**Pull request merged**